### PR TITLE
[v10.6.0] Only Require One Valid Dispenser in Validation Logic

### PR DESCRIPTION
**Pull request by ouziel-slama**
* [x] Double-check the spelling and grammar of all strings, code comments, etc.
* [x] Double-check that all code is deterministic that needs to be
* [x] Add tests to cover any new or revised logic
* [x] Ensure that the test suite passes
* [x] Update the project [release notes](release-notes/)
* [x] Update the project documentation, as appropriate, with a corresponding Pull Request in the [Documentation repository](https://github.com/CounterpartyXCP/Documentation)

### CODE REVIEW

File: counterparty-core/counterpartycore/protocol_changes.json [hunk 1]
```diff
@@ -637,5 +637,12 @@
         }
       }
     }
+  },
+  "accept_only_one_valid_dispenser": {
+    "minimum_version_major": 11,
+    "minimum_version_minor": 1,
+    "minimum_version_revision": 0,
+    "block_index": 966000,
```
Comment (adamkrellenstein) [inline counterparty-core/counterpartycore/protocol_changes.json:645]: Let's make this for one week from today.

**Review by adamkrellenstein — APPROVED** [review]
Need release notes!

**Review by adamkrellenstein — CHANGES_REQUESTED** [review]
and tests for these cases

File: counterparty-core/counterpartycore/protocol_changes.json (hunk 1 above)
Comment (ouziel-slama) [reply counterparty-core/counterpartycore/protocol_changes.json]: "block_index": 868200

**Review by adamkrellenstein — APPROVED** [review]

**Pull request merged**